frontends/web/src/locales/en/app.json

```diff
@@ -462,7 +482,6 @@
               "title": "Fees for Buying"
             },
             "intro": "European buying of cryptocurrencies with EUR.",
-            "learnmore": "Learn more about BTC Direct",
```

Comment:
just fyi: this is not yet in locize.

moved "learnmore" key one level up so as it is the same text for "learn more" regardless where the link is.

but later I realized that there are a few small inconsitencies how all this  guide text is structured.